ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -246,6 +250,20 @@ l2leaf:
     rack: "rackE"
     mlag_peer_l3_ipv4_pool: 10.255.251.0/24
     mlag_peer_ipv4_pool: 10.255.252.0/24
+    uplink_structured_config:
+      storm_control:
+        broadcast:
+          level: 25
+        unknown_unicast:
+          level: 25
+      native_vlan: tag
```

Comment:
Maybe let's use an actual number here for the tests otherwise it generates some quite weird configuration (I know these are only tests but they may be used as examples by users)

Reply:
This is indeed a valid command, as per our chat, let's create a PR to update the documentation in `eos_cli_config_gen`.

Reply:
I don't agree with this
the eos config `switchport native vlan tag` cannot _replace_ `switchport native vlan 123`. It should be _another_ line in the config. So we need a new key `native_vlan_tag: < true | false >`

Reply:
@gmuloc @carlbuchmann

Reply:
indeed agreeing with @ClausHolbechArista  - I did not realise both where possible

Reply:
My bad, EOS only accept one or the other. But I still think we need to do something, since we should not mix types between integer and string.

Reply:
actually double checking on CLI it seems that tag and a given ID are mutually exclusive.